app/sql.js

```diff
@@ -421,6 +424,14 @@ async function updateToSchemaVersion6(currentVersion, instance) {
      ADD COLUMN friendRequestStatus INTEGER;`
   );
 
+  await instance.run(
+    `CREATE TABLE lastHashes(
+      snode STRING PRIMARY KEY,
```

Comment:
Didn't we previously have issues with STRING being loosely stored as a number sometimes?

Reply:
Ah yes I was going to ask you about this, was there any reason to use STRING over TEXT? Or should we just use TEXT all the time?
In this instance I don't *think* the number problem because the snode string will always end in `.snode`, but maybe I should just use TEXT anyway?

Reply:
Locking in text